### DISCUSSION TITLE
Tripeptides Data for MACE-OFF Model

### DISCUSSION THREAD

**Discussion by tkreiman** [ACEsuit/mace#541] [Q&A]
We are very interested in the MACE-OFF paper. We have downloaded the data but noticed that there is no data corresponding to the tripeptides results reported in the paper. Is this data available?

Thanks for the help.

**Comment by gabor1** [comment]
@jharrymoore can you help here?

**Reply by tkreiman** [reply]
Just wanted to follow up on this to see if this data was available?

**Reply by gabor1** [reply]
Sorry - I didn't follow up with Harry on this, and he is currently on holiday, returning two weeks from now.

**Reply by wiederm** [reply]
any update on this?

**Reply by jharrymoore** [reply]
Hi, apologies for the delay, I believe David will have had the originals, to which I no longer have access, but the energy and force labels [here](https://drive.google.com/file/d/1JfeTng7O8VmKYVUnb9Yy4pxvrtYOGnFT/view?usp=sharing)  should be the wb97M reference values you are looking for
